ports/tftp-hpa/fix-build-gcc-10.patch

```diff
@@ -0,0 +1,24 @@
+diff -urN tftp-hpa-5.2.orig/tftp/main.c tftp-hpa-5.2/tftp/main.c
+--- tftp-hpa-5.2.orig/tftp/main.c	2020-11-14 22:21:15.851650899 -0700
++++ tftp-hpa-5.2/tftp/main.c	2020-11-14 22:21:41.878327755 -0700
+@@ -95,7 +95,7 @@
+ int margc;
+ char *margv[20];
+ const char *prompt = "tftp> ";
+-sigjmp_buf toplevel;
++static sigjmp_buf toplevel;
```

Comment:
Could also wrap the static keyword in ifdef __QNX__ statements for upstreaming later, thought it would be ugly.

Reply:
This is not QNX specific. This patch is taken from Alpine.